tools/schema-explorer/static/css/style.css

```
@@ -0,0 +1,1356 @@
+   stylesheet to avoid leaking Bootstrap's global heading/body rules into
+   Material. Keep these selectors under the explorer roots. */
+.schema-spa-host,
+.schema-embed {
+  --se-body-bg: #fff;
+  --se-border-color: #dee2e6;
+  --se-muted: #6c757d;
+  --se-light-bg: #f8f9fa;
+  --se-success: #198754;
+  --se-danger: #dc3545;
+  --se-warning: #ffc107;
+  box-sizing: border-box;
```

Stylelint (17.13.0)

[error] Expected empty line before declaration (declaration-empty-line-before)

(declaration-empty-line-before)